About.html

```diff
@@ -0,0 +1,99 @@
+            <p class="punchline2">Anyone who follows the crypto space knows that non-fungible tokens (NFTs) are all the
+                rage of late.At a very high level, most NFTs are part of the Ethereum blockchain. Ethereum is a
+                cryptocurrency, like bitcoin or dogecoin, but its blockchain also supports these NFTs, which store extra
+                information that makes them work differently from, say, an ETH coin. It is worth noting that other
+                blockchains can implement their own versions of NFTs.</p>
+            <div class="endline">Please contact us for any furthur question about NFTs contact us at <a
+                    href="#">anicetmurhula20@gmail.comn</a></div>
+        </div>
+
+        <div class="about-content">
+            <div class="about-content-title">Non-fungible token (NFT)</div>
+            <p>A non-fungible token is a non-interchangeable unit of data stored on a blockchain, a form of
+                digital ledger, that can be sold and traded.[1] Types of NFT data units may be associated with digital
+                files such as photos, videos, and audio. Because each token is uniquely identifiable, NFTs differ from
+                blockchain cryptocurrencies, such as Bitcoin.
+
+                NFT ledgers claim to provide a public certificate of authenticity or proof of ownership, but the legal
+                rights conveyed by an NFT can be uncertain. NFTs do not restrict the sharing or copying of the
+                underlying digital files, do not necessarily convey the copyright of the digital files, and do not
+                prevent the creation of NFTs with identical associated files.</p>
+                <div class="about-img"><img src="./images/aboutImg1.jpg" alt=""></div>
+        </div>
+        <hr>
+        <div class="about-past">
+            <div class="past-title">See the three last NFTs created</div>
+            <p>Take a look on our best products NFTs created</p>
+            <div class="bestNFS">
+                <div class="best1 best">Best art of 2015</div>
+                <div class="best2 best">Best art of 2020</div>
+                <div class="best3 best">Best art 2022</div>
+            </div>
+        </div>
+    </main>
+
+    <footer>
+        <h2 class="footer-line-1">Partners</h2>
+        <div class="footer-list-container">
+            <div class="for-nav"><img src="./images/NflLogo.png" alt="Here is the Nfl company logo"></div>
+            <div class="for-nav"><img src="./images/NBA.jpg" alt="Here is the NBA company logo"></div>
+            <div class="for-nav"><img src="./images/ufc.jpg" alt="Here is the ufc company logo"></div>       
+        </div>
+
+        <ul>
+            <li>Polygon</li>
+            <li>NFL</li>
+            <li>NBA</li>
+            <li>UFC</li>
+            <li>coinbase</li>
+            <li>Fantomas</li>
+            <li>Micielo Hills</li>
+        </ul>
+    </footer>
```

Comment:
- [ ] The footer section of the about page does not have a partners' section. As seen below, I am suggesting the you kindly make the footer match the original design

![image](https://user-images.githubusercontent.com/4247740/158914960-c6284cab-1234-4505-ace6-ae4cb6d73aca.png)